ci/39-numba.yaml

```diff
@@ -16,3 +16,10 @@ dependencies:
   - matplotlib
   # optional
   - geopandas
+  - pygeos>=0.10
+  - sphinx>=1.4.3
+  - sphinxcontrib-bibtex==1.0.0
+  - sphinx_bootstrap_theme
+  - numpydoc
+  - nbsphinx
+  - pygeos
```

Comment:
`pygeos` duplicated.

Reply:
I noticed that `pygeos` is not included in the [`37-numba.yaml`](https://github.com/pysal/esda/blob/master/ci/37-numba.yaml) environment. Should it be?